test/test.construct.ts

```diff
@@ -477,7 +478,18 @@ export = {
         /Cannot determine default child for . There is both a child with id "Resource" and id "Default"/);
       test.done();
 
-    }
+    },
+    'constructs created in an Aspect are prepared'(test: Test) {
+      const root = new Root();
+      const construct = new MyBeautifulConstruct(root, 'BeautifulConstruct');
```

Comment:
This construct confuses the issue for me. Can't we get rid of this and just apply on the root? Or at least change the type of this one to a regular `Construct`? We're not testing that this construct participates in the `prepare()` phase, right?

Reply:
will do, might have gotten carried away with the constructs inner world here.

I'll move it to a regular construct